features/main_landing_page.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Main landing page
+  As a visitor
+  In order to get an overview over the service
+  I need to see a main landing page
+
+  Scenario: Visitor visits main page
+    When I visit the main landing page
+    Then I should see "Welcome to HomeChef"
+    And I should see input of type "text" with name "enter_location"
+    And I should see a placeholder for dishes in general with an id of "random_dishes"
```

Comment:
I would use `I should see a "random_dishes" placeholder`. This feels like a duplicate.